### PR TITLE
Load chemkin bug fix - extended elements and digits

### PR DESCRIPTION
**Pull request by davidfarinajr**
### Motivation or Problem
I tried to load RMG generated chemkin file, but errors were raised when
1) trying to read thermo block with extended elemental composition syntax
2) reading a reaction with a species the has a digit as its first character (i.e. 2-BTP)

### Description of Changes
1) check for extended syntax notation, and append the elemental comp. line to thermo block
2) check species dictionary before assuming that the leading digit corresponds to stoic coefficient

### Testing
I was able to load my chemkin files

### CODE REVIEW

**Review by ChrisBNEU — APPROVED** [review]
these look good, thanks for making them.

File: rmgpy/chemkin.pyx [hunk 1]
```diff
@@ -1227,7 +1237,7 @@ def read_thermo_block(f, species_dict):
                     logging.warning('Skipping unexpected species "{0}" while reading '
                                     'thermodynamics entry.'.format(label))
             thermo_block = ''
-        if len(thermo_block.split('/n')) > 4:
+        if len(thermo_block.split('/n')) > 5:
             raise ChemkinError('Should only have 4 lines in a thermo block:\n{0}'.format(thermo_block))
```
Comment (ChrisBNEU) [inline rmgpy/chemkin.pyx:1241]: nitpicky, but maybe change error message text: 
```suggestion
            raise ChemkinError('Should only have a maximum of 5  lines in a thermo block:\n{0}'.format(thermo_block))
```

**Review by ChrisBNEU — CHANGES_REQUESTED** [review]
I missed this before, but we should change the message text. Otherwise this is good I think.

File: rmgpy/chemkin.pyx (hunk 1 above)
Comment (davidfarinajr) [reply rmgpy/chemkin.pyx]: Thanks Chris! Just fixed this

**Review by ChrisBNEU — APPROVED** [review]
looks good!

**Pull request merged**